with Raspberry

```diff
@@ -0,0 +1,56 @@
+**Zenbot Installation guide on Raspberry (Pi3 / Pi4)
```

Comment:
did you mean to update https://github.com/DeviaVir/zenbot/blob/unstable/docs/installation/raspberrypi.md ?

Reply:
i have seen some docs about Zenbot can not run in Raspberry 1GB, but i follow this steps and its worked for me.
so i thought to share with this steps.